projects/noloan/common/src/main/java/hasadna/noloan/common/FirebaseAuthontication.java

```diff
@@ -0,0 +1,43 @@
+package hasadna.noloan.common;
+
+import android.util.Log;
+
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.FirebaseUser;
+
+import java.util.List;
+
+public class FirebaseAuthontication {
+  private static final String TAG = "FirebaseAuthontication";
+
+  private static FirebaseAuthontication instance;
+  private FirebaseUser user;
+  private FirebaseAuth auth;
+
```

Comment:
So this is a Singleton ?

Perhaps mention that in a comment, as you did with the `SmsMessages`.